### PR TITLE
[Azure.Core] Update SerializableOptions.cs

### PR DESCRIPTION
**Pull request by nisha-bhatia**
Fixes [#35851](https://github.com/Azure/azure-sdk-for-net/issues/35851)
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by nisha-bhatia** [comment]
https://github.com/Azure/azure-sdk-for-net/issues/35851

File: sdk/core/Azure.Core/src/SerializableOptions.cs [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
-    /// TODO
+    /// Provides the client options for serializing models.
     /// </summary>
     public class SerializableOptions
     {
         /// <summary>
         /// TODO
         /// </summary>
-        public bool IncludeReadOnlyProperties { get; set; } = true;
+        public bool IgnoreReadOnlyProperties { get; set; } = false;
```
Comment (m-nash) [inline sdk/core/Azure.Core/src/SerializableOptions.cs:14]: nit: the `= false` is not needed any  longer since that is the default for a bool

**Review by m-nash — APPROVED** [review]

**Pull request merged**